bugsnag-kmp/src/appleMain/kotlin/com/bugsnag/kmp/Bugsnag.kt

```diff
@@ -22,8 +22,19 @@ public actual object Bugsnag {
 
     public actual fun isStarted(): Boolean = PlatformBugsnag.isStarted()
 
-    public actual fun notify(error: Throwable) {
-        PlatformBugsnag.notify(BugsnagNSException(error))
+    @Suppress("UNCHECKED_CAST")
```

Comment:
this doesn't appear to be an unchecked cast in the implementation of this function, so this annotation can be removed